kyo-combinators/shared/src/main/scala/kyo/AsyncCombinators.scala

```diff
@@ -41,8 +41,8 @@ extension [A, E, Ctx](effect: A < (Abort[E] & Async & Ctx))
     @targetName("zipRightPar")
     def &>[A1, E1, Ctx1](next: A1 < (Abort[E1] & Async & Ctx1))(
         using
-        i1: Isolate.Contextual[Ctx, Sync],
-        i2: Isolate.Contextual[Ctx1, Sync],
+        i1: Isolate[Ctx, Sync, Any],
+        i2: Isolate[Ctx1, Sync, Any],
```

Comment:
these methods could be more flexible with `Isolate[Ctx1, Sync, Any]`. I'm planning to follow up on it